examples/package.json

```diff
@@ -1,6 +1,9 @@
 {
     "private": true,
     "dependencies": {
+        "@angular/core": "14",
```

Comment:
Aren't these at 16 something now?

Reply:
nope, not according to https://www.npmjs.com/package/@angular/compiler-cli